src/modules/CartPage/CartItem/CartItem.tsx

```diff
@@ -1,42 +1,119 @@
 import React from 'react';
+
+import cn from 'classnames';
+// import { useAppSelector } from '../../../store/hooks';
 import { Phone } from '../../../types/Phone';
 import styles from './CartItem.module.scss';
+import { ReactComponent as CloseIcon }
+  from '../../../static/icons/close_icon.svg';
+import { ReactComponent as MinusIcon }
+  from '../../../static/icons/minus_icon.svg';
+import { ReactComponent as PlusIcon }
+  from '../../../static/icons/plus_icon.svg';
 
 type Props = {
   phone: Phone | null,
 };
 
 export const CartItem: React.FC<Props> = ({ phone }) => {
+  // const isThemeDark = useAppSelector((state) => state.theme);
+  const isThemeDark = true;
+  const isMinusDisabled = true;
+  const isPlusDisabled = false;
+
+  const getIconColor = (isTheme: boolean, isDisabled: boolean) => {
+    if (isTheme) {
+      return isDisabled ? '#4A4D58' : '#F1F2F9';
+    }
+
+    return isDisabled ? '#B4BDC3' : '#0F0F11';
+  };
+
+  const getClass = (isTheme:boolean, isDisabled: boolean) => {
+    if (isTheme) {
+      return isDisabled
+        ? styles.changeAmountButton__dark
+        : styles.changeAmountButton__activeDark;
+    }
+
+    return isDisabled
+      ? {}
+      : styles.changeAmountButton__active;
+  };
+
+  const minusIconColor = getIconColor(isThemeDark, isMinusDisabled);
+  const plusIconColor = getIconColor(isThemeDark, isPlusDisabled);
+  const minusClass = getClass(isThemeDark, isMinusDisabled);
+  const plusClass = getClass(isThemeDark, isPlusDisabled);
```

Comment:
Need to think a little bit about alternative solutions, because it doesn't look good so far)